tardis/montecarlo/src/rpacket.c

```diff
@@ -15,9 +15,11 @@ rpacket_init (rpacket_t * packet, storage_model_t * storage, int packet_index,
   double current_nu = storage->packet_nus[packet_index];
   double current_energy = storage->packet_energies[packet_index];
   double current_mu = storage->packet_mus[packet_index];
-  double comov_current_nu = current_nu;
+//  double comov_current_nu = current_nu;
   int current_shell_id = 0;
   double current_r = storage->r_inner[0];
+  double comov_current_nu = current_nu * (1 - (current_mu * current_r * storage->inverse_time_explosion * INVERSE_C)); 
+/*  
```

Comment:
What do you mean? Its exactly like you told me to change it. This is part of handling the CMF - LF initialization of the BB field at the photosphere, which we had to to to match the MC. Discussion [here](https://gitter.im/tardis-sn/tardis?at=579b83b0f1da4f376e186512) (hopefully)